src/tokens/RwaTokenFactory.t.sol

```diff
@@ -25,52 +25,38 @@ import {RwaTokenFactory} from "./RwaTokenFactory.sol";
 contract RwaTokenFactoryTest is DSTest {
     uint256 internal constant WAD = 10**18;
 
-    ForwardProxy internal op;
-    ForwardProxy internal recipient;
-    RwaTokenFactory internal tokenFactory;
-    string internal constant NAME = "RWA001-Test";
-    string internal constant SYMBOL = "RWA001";
+    ForwardProxy recipient;
+    RwaTokenFactory tokenFactory;
+    string NAME = "RWA001-Test";
+    string SYMBOL = "RWA001";
```

Comment:
We should probably keep the `internal` and `constant` modifiers to avoid compiler warnings.

It's not big deal, but if we get too many compiler warnings, we tend to create the habit to ignore them and that can lead to us missing something important down the road.